config/routes.rb

```diff
@@ -9,11 +9,17 @@
   resources :user_sessions
   resources :users
 
+  resources :decks
+
   get 'login' => 'user_sessions#new', :as => :login
   post 'logout' => 'user_sessions#destroy', :as => :logout
 
+  get 'home' => "home#index"
+
+  put 'decks' => 'profiles#set_current_deck', :as => :set_current_deck
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.